### PR TITLE
Remove ContainerService cmdlets and documentation

### PR DESCRIPTION
**Pull request by Sandido**
The Container Service API was deprecated in January 2020: https://azure.microsoft.com/en-us/updates/azure-container-service-will-retire-on-january-31-2020/
The cmdlets have not been functional except for Get operations. I discussed this change with Damien and this removal is not a breaking change. 

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Compute/Compute/ChangeLog.md [hunk 1]
```diff
@@ -20,6 +20,7 @@
 
 -->
 ## Upcoming Release
+* Removed all `ContainerService` cmdlets. The Container Service API was deprecated in January 2020.
```
Comment (VeryEarly) [inline src/Compute/Compute/ChangeLog.md:23]: I suggest to list all removed cmdlets by name here, to make it easier for users to recognize what were removed.

**Review by VeryEarly — CHANGES_REQUESTED** [review]
please also suppress breaking change issues in:
tools/StaticAnalysis/Exceptions/Az.Compute/BreakingChangeIssues.csv

**Comment by grizzlytheodore** [comment]
@VeryEarly  your comment has been addressed. please review

**Comment by Sandido** [comment]
It doesn't look like the breaking change suppressions were added.

**Comment by Sandido** [comment]
Added breaking change suppressions.

**Review by VeryEarly — APPROVED** [review]

**Pull request merged**